frontend/src/app/data/transactions.service.ts

```diff
@@ -7,6 +7,9 @@ import Transaction from 'src/app/model/Transaction';
   providedIn: 'root'
 })
 export class TransactionsService {
+  public static getSignedAmount(transaction: Transaction): Number {
+    return transaction.direction === 'RECEIVE' ? transaction.amount : -transaction.amount;
```

Comment:
`Math.abs()`?

Reply:
No, it's not absolute value. Quite the opposite, it adds a sign to a postive value (100 PAY becomes +100 and 200 RECEIVE becomes -200)

Reply:
And I left this logic here having added it to SignedAmountPipe by mistake. It's not used anymore, so I'll remove it.

Reply:
Nice